### PR TITLE
Checkbox

### PR DESCRIPTION
**Pull request by kaitmore**

### CODE REVIEW

**Comment by shindigira** [comment]
@kaitmore I think we are all using `prettier` with single quotes which autoformats on-save.

I believe this is the vscode version of prettier: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

**Comment by tilleryd** [comment]
@shindigira The single vs double quote just spawned another discussion on the subject that we never came to a conclusion on.  I think the important thing is that each project has a prettier config in `package.json` so editor settings don't take precedence.  It seems kind of silly on our part to not have a consensus on which quote we go with.  On one hand Doug Crockford, a core JS maintainer and creator or JSLint, uses double while AirBnB's style guide uses single.


Doug Crockford (JSLint) https://plus.google.com/+%D0%95%D0%B2%D0%B3%D0%B5%D0%BD%D0%B8%D0%B9%D0%9E%D1%80%D0%B5%D1%85%D0%BE%D0%B2/posts/dykmroX6oYH

AirBnB style guide
https://github.com/airbnb/javascript#strings--quotes

what the meme

File: src/stories/Checkbox.story.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+stories.addDecorator(withKnobs);
+
+stories.add("standard", () => {
+  return (
+    <div style={{ width: "50vw" }}>
+      <Checkbox label={text("label", "this is a checkbox")} />
+    </div>
+  );
+});
```
Comment (shindigira) [inline src/stories/Checkbox.story.js]: May also want to include this [checkbox story](https://gist.github.com/shindigira/500700b4acc7d1c7c6e97b262514ab2c)

Also, may want to have prop-type checks for `onChange` and `defaultChecked` optional.

File: src/components/Checkbox/Checkbox.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  > input {
+    margin: 0 8px;
+  }
+`;
+
+//pick out label, all other props are passed to checkbox element
+const Checkbox = ({ label, ...props }) => {
+  return (
+    <Label>
+      <Box {...props} />
```
Comment (shindigira) [inline src/components/Checkbox/Checkbox.js]: Nice, will accept all attrs or props.

File: src/stories/Checkbox.story.js (hunk 1 above)
Comment (kaitmore) [reply src/stories/Checkbox.story.js]: done, thanks!

**Pull request merged**